applications/fax/src/fax_smtp.erl

```diff
@@ -280,6 +280,7 @@ maybe_system_report(#state{faxbox='undefined', account_id='undefined'}=State) ->
         'false' -> 'ok'
     end;
 maybe_system_report(State) ->
+    send_outbound_smtp_fax_error(State),
```

Comment:
@jamesaimonetti fax attachment error will be sent here. this is not desirable. you can put behind a configuration flag (false by default) and would prefer to put that in the `maybe_faxbox_log` clauses